shiny_app/global.R

```diff
@@ -241,16 +241,16 @@ HCWdata_list_data_tab <- c ("Specialist Cancer Wards and Treatment Areas" = "Hea
 
 
 #extra choices for data tables
-data_list_data_tab <- c("Positive Cases" = "LabCases",
-                        "Positive Cases by age and sex" = "LabCases_AgeSex",
-                        "Positive Cases by deprivation" = "LabCases_SIMD",
-                        "Distribution of confirmed COVID-19 cases by age group" = "Cases_AgeGrp",
+data_list_data_tab <- c("Positive PCR Cases" = "LabCases",
+                        "Positive PCR Cases by age and sex" = "LabCases_AgeSex",
+                        "Positive PCR Cases by deprivation" = "LabCases_SIMD",
```

Comment:
Good point, this was the style that was already there so I didn't change it but it could probably do with being universally updated. Thanks for the review!